install/installing-with-kata-manager.md

```diff
@@ -8,29 +8,39 @@
 ## Introduction
 `kata-manager` automates the Kata Containers installation procedure documented for [these Linux distributions](README.md#supported-distributions).
 
-> Note:
+> **Note**:
+> - `kata-manager` requires `curl` and `sudo` installed on your system.
+>
 > - Full installation mode is only available for Docker container manager. For other setups, you
 > can still use `kata-manager` to [install Kata package](#install-kata-packages-only), and then setup your container manager manually.
+>
+> - You can run `kata-manager` in dry run mode by passing the `-n` flag. Dry run mode allows you to review the
+> commands that `kata-manager` would run, without doing any change to your system.
+
 
 ## Full Installation
 This command does the following:
 1. Installs Kata Containers packages
 2. Installs Docker
 3. Configure Docker to use the Kata OCI runtime by default
 
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) install-docker-system"
 ```
-$ bash -c "$(curl -fsSL \
-$ https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
- install-docker-system"
-```
 
+<!--
```

Comment:
bleh, the first time we have buried HTML in our mardown documents I believe.....